config/initializers/cors.rb

```
@@ -0,0 +1,12 @@
+Rails.application.config.middleware.insert_before 0, Rack::Cors do
+  allow do
+    origins ->(source, _env) {
```

RuboCop (1.86.1)

[convention] Use the `lambda` method for multiline lambdas.

(Style/Lambda)